de.tu_bs.cs.isf.e4cf.core/src/de/tu_bs/cs/isf/e4cf/core/stringtable/E4CEventTable.java

```diff
@@ -16,4 +16,5 @@
 	
 	public static final String EVENT_FILTER_CHANGED = "EVENT_FILTER_CHANGED";
 	public static final String EVENT_VIEW_TOGGLE = "EVENT_VIEW_TOGGLE";
+	public static final String EVENT_SET_TREE_SELECTION = "EVENT_SET_TREE_SELECTION";

```

Comment:
This Event seems to be unused?

It would explain why the selection is visually lost in the ProjectTree while it is still active in the selection service. Messing with handler CanExecute-Functions.

This update must be done in all handlers.

Reply:
removed it